mlir/lib/Dialect/SPIRV/Transforms/UpdateVCEPass.cpp

```diff
@@ -174,6 +174,22 @@ void UpdateVCEPass::runOnOperation() {
   if (walkResult.wasInterrupted())
     return signalPassFailure();
 
+  // Update min version requirement for capabilities after deducing them
+  for (auto &cap : deducedCapabilities) {
```

Comment:
```suggestion
  for (spirv::Capability cap : deducedCapabilities) {
```
See https://llvm.org/docs/CodingStandards.html#use-auto-type-deduction-to-make-code-more-readable.
Also, no need to use references for enums.

Reply:
Done.